src/lib/snark_worker/events.ml

```diff
@@ -0,0 +1,49 @@
+open Core
+
+(*FIX: register_event fails when adding base types to the constructors*)
```

Comment:
Can you be more descriptive? Should there be an issue to keep track of this?

Reply:
Ah, it seems to be from Functor.ml.

Reply:
Yeah, I don't know what's with this as well